lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -0,0 +1,346 @@
+<?php
+/**
+ * Manage and give access to lizmap configuration.
+ *
+ * @author    3liz
+ * @copyright 2012 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+
+use Lizmap\Project;
+
+/**
+ * @deprecated
+ * @FIXME getXml, getComposer
+ * Verify this methods are not used in external modules so we can delete them without risk, otherwise, we have to implement them
+ * in Project and call it here
+ */
+class lizmapProject
+{
+    /**
+     * @var Project\Project
+     */
+    protected $proj;
+
+    /**
+     * constructor.
+     *
+     * @param string           $key      : the project name
+     * @param LizmapRepository $rep      : the repository
+     * @param mixed            $context
+     * @param mixed            $services
+     */
+    public function __construct($key, $rep, $context, $services)
+    {
+        try {
+            $this->proj = new \Lizmap\Project\Project($key, $rep, $context, $services);
+        } catch (Project\UnknownLizmapProjectException $e) {
+            throw $e;
+        }
+    }
+
+    public function clearCache()
+    {
+        $this->proj->clearCache();
+    }
+
+    public function getQgisPath()
+    {
+        return $this->proj->getQgisPath();
+    }
+
+    public function getRelativeQgisPath()
+    {
+        return $this->proj->getRelativeQgisPath();
+    }
+
+    public function getKey()
+    {
+        return $this->proj->getKey();
+    }
+
+    public function getRepository()
+    {
+        return $this->proj->getRepository();
+    }
+
+    public function getFileTime()
+    {
+        return $this->proj->getFileTime();
+    }
+
+    public function getCfgFileTime()
+    {
+        return $this->proj->getCfgFileTime();
+    }
+
+    public function getProperties()
+    {
+        return $this->proj->getProperties();
+    }
+
+    public function getOptions()
+    {
+        return $this->proj->getOptions();
+    }
+
+    public function getLayers()
+    {
+        return $this->proj->getLayers();
+    }
+
+    public function getLayer($layerId)
+    {
+        return $this->proj->getLayer($layerId);
+    }
+
+    public function getXmlLayer($layerId)
```

Comment:
This methods was not in the original class

Reply:
It was in qgisProject